utils/create_test_avatar.py

```diff
@@ -0,0 +1,20 @@
+import tempfile
```

Comment:
This is not used. Please remove along with directory.

Reply:
below uses it. Tenth line

Reply:
Apologies for not being specific. This file is not used. I removed it, tests pass. Please remove the file along with directory.